aten/src/ATen/native/cuda/ReflectionPadding1d.cu

```diff
@@ -0,0 +1,215 @@
+#include <ATen/ATen.h>
+#include <ATen/cuda/CUDAApplyUtils.cuh>
+#include <ATen/cuda/CUDAContext.h>
+#include <ATen/NativeFunctions.h>
+#include <ATen/TensorUtils.h>
+#include <ATen/Utils.h>
+// keeping THC headers for atomicAdd
+#include <THC/THCGeneral.h>
```

Comment:
is it sufficient to just include THCAtomics?

Reply:
Let me try.